BPMRFFEApp/Db/bpmrffe.db

```diff
@@ -0,0 +1,305 @@
+# BPM RFFE Database records
+
+# Attenuation
+
+record(ao, "$(P)$(R)RFFEAtt-SP") {
+    field(DESC, "Set attenuation")
+    field(DTYP, "stream")
+    field(PINI, "YES")
+    field(SCAN, "Passive")
+    field(OUT, "@bpmrffe.proto outAtt $(PORT)")
+    field(EGU, "dB")
+    field(FLNK, "$(P)$(R)RFFEAtt-RB.PROC CA")
+}
```

Comment:
I've found [a description of a Python utility script in the firmware](https://github.com/lnls-dig/rffe-uc-fw/blob/master/utils/rffe_nuttx_lib.py#L115) that specifies that the attenuation should be in the range 0 up to 31.5dB (0.5 step size). Should that be limited by the IOC (even though it appears not to be really limited by the firmware)?

Reply:
The readback value really shows the updated value. However, this value may be something like `1000` (maybe any value?). This will be written simply accepted by the firmware and written in its `/dev/att0` (and `/dev/feram0`) file and [this will be sent to the attenuation hardware](https://github.com/lnls-dig/rffe-uc-fw/blob/21329d8f9be430d373375f76a7168d4c50047cb2/rffe-board/src/lpc17_40_appinit.c#L267) (without any further checking as far as I understand, which is not much).

Does the hardware know how to deal with a input of 1000dB? Is it rejecting it and notifying the firmware somehow? Is it acceptable for us to have an attenuation value of 1000dB apparently (or really) set in the device (visible to the IOC in its RB)?

Reply:
The hardware will truncate the MSB outside the range of the attenuator, so it is a good idea to range limit to 31.5dB, 0.5 dB step size. There is no check from the firmware side, but it this could be improved in the future to avoid silent errors.

Reply:
To simplify our deploy scope for this week, I've only limited the attenuation to the `[0, 31.5]dB` range. Step size is not easily configurable as far as I know. I've created an issue (#4) to handle this in a future release.